From 9734b82288a429d3cb04d0c3e1f55b25447b5d51 Mon Sep 17 00:00:00 2001
Date: Fri, 22 Jan 2021 16:34:29 +0800
Subject: dismiss company_name info of java -version

Summary: <version>: <dismiss company_name info of java -version>
LLT: NA
Bug url: NA
---
 jdk/src/share/classes/sun/misc/Version.java.template | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/jdk/src/share/classes/sun/misc/Version.java.template b/jdk/src/share/classes/sun/misc/Version.java.template
index 022c14281..ffdaf9fbc 100644
--- a/jdk/src/share/classes/sun/misc/Version.java.template
+++ b/jdk/src/share/classes/sun/misc/Version.java.template
@@ -45,7 +45,7 @@ public class Version {
         "@@java_runtime_version@@";
 
     private static final String company_name =
-        "@@company_name@@";
+        "";
 
     static {
         init();
-- 
2.19.0

